services/blockchain-indexer/shared/dataService/business/validator/validateBLSKey.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { requestConnector } = require('../../../utils/request');
+
+const validateBLSKey = async params => {
+	const { blsKey, proofOfPossession } = params;
+
+	const response = {
+		data: {},
+		meta: {},
+	};
+
+	const { valid } = await requestConnector('validators_validateBLSKey', {
```

Comment:
Please create an interface `validateBLSKey` in the connector and invoke the same.
```suggestion
	const response = {
		data: {},
		meta: {},
	};

	const { blsKey, proofOfPossession } = params;
	const { valid } = await requestConnector('validateBLSKey', {
```

Reply:
applied.